src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/SafeBufferTests.cs

```diff
@@ -25,11 +25,11 @@ public void Initialize_InvalidNumBytes_ThrowsArgumentOutOfRangeException()
         }
 
         [Fact]
-        public void Initialize_NumBytesTimesSizeOfEachElement_ThrowsOverflowException()
+        public void Initialize_NumBytesTimesSizeOfEachElement_ThrowsArgumentOutOfRangeException()
         {
             var buffer = new SubBuffer(true);
-            Assert.Throws<OverflowException>(() => buffer.Initialize(uint.MaxValue, uint.MaxValue));
-            Assert.Throws<OverflowException>(() => buffer.Initialize<int>(uint.MaxValue));
+            Assert.Throws<ArgumentOutOfRangeException>(() => buffer.Initialize(uint.MaxValue, uint.MaxValue));
+            Assert.Throws<ArgumentOutOfRangeException>(() => buffer.Initialize<int>(uint.MaxValue));
```

Comment:
Presumably like most tests in the corefx repo these also run against netfx, and netfx will still be throwing OverflowException?  If so, you'll need to specialize this with AssertExtensions.Throws, which has an overload that allows you to specify which exception is expected for core and which is expected for netfx.